chromeapp/test/scripts/dnssd/dns-packet-test.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
```

Comment:
Exposed it on the public API (I'd kept it internal for some reason) and added tests.

done